README.md

```diff
@@ -16,57 +16,112 @@ Docker images can be found in Expedia Group's [dockerhub](https://hub.docker.com
 
-Beekeeper comprises two separate Spring-based Java applications. One application schedules paths for deletion in a shared database, and the other performs deletions.
+Beekeeper comprises three separate Spring-based Java applications. One application schedules paths and metadata for deletion in a shared database, with one table for unreferenced paths and another for expired metadata. The other two applications perform deletions for the paths and metadata, respectively.
 
-## End-to-end lifecycle example
+## Beekeeper Architecture
 
-![Beekeeper architecture](.README_images/architecture_diagram.png)
+![Beekeeper architecture](.README_images/updated_architecture_diagram.png)
 
+## Unreferenced paths
+The unreferenced property can be added to tables to monitor when paths become unreferenced, e.g. when a table or partition is dropped. 
+
+End-to-end lifecycle example
 1. A Hive table is configured with the parameter `beekeeper.remove.unreferenced.data=true` (see [Hive table configuration](#hive-table-configuration) for more details.)
 2. An operation is executed on the table that orphans some data (alter partition, drop partition, etc.)
-3. Hive Metastore events are emitted by the [Hive Metastore Listener](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-listener) as a result of the operation.
-4. Hive events are picked up from the queue by Beekeeper using [Apiary Receiver](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-receivers).
-5. Beekeeper processes these messages and schedules orphaned paths or expired tables for deletion by adding them to a database.
-6. The scheduled paths or tables are deleted by Beekeeper after a configurable delay, the default is 3 days for paths and 30 days for tables (see [Hive table configuration](#hive-table-configuration) for more details.)
+3. Hive MetaStore events are emitted by the [Hive MetaStore Listener](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-listener) as a result of the operation.
+4. Hive events are picked up from the queue by Beekeeper using the [Apiary Receiver](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-receivers).
+5. Beekeeper processes these messages and schedules orphaned paths for deletion by adding them to a database.
+6. The scheduled paths are deleted by Beekeeper after a configurable delay, the default is 3 days (see [Hive table configuration](#hive-table-configuration) for more details.)
+
+## Time To Live, TTL 
+There is also a TTL property which, when added, will delete tables and their locations after a configurable delay. If no delay is specified the default is 30 days. 
```

Comment:
Can you word this sentence more like the starting sentence of the previous section so they are consistent. Something like The "expired" property can be added to tables to enforce a Time To Live (TTL) and will delete tables and...  etc.